its/ruling/src/test/resources/guava/java-S5803.json

```diff
@@ -0,0 +1,10 @@
+{
+'com.google.guava:guava:src/com/google/common/io/Closer.java':[
+252,
```

Comment:
This one looks like a FP to me (while a bit borderline), we are inside a class itself annotated with `@VisibleForTesting`, so it feels to me that it's OK to do call a something which is also annotated with `VisibleForTesting`. What's your opinion of it?

Reply:
I think we should exclude those members, is the owner is annotated @VisibleForTesting too

Reply:
I thought about it and I think we should report this issue. The 'Logger', which is used inside 'Closer' and 'Closer' are in different files, so if 'Logger' is marked as private, it is not visible in 'Closer'

Reply:
Fine by me!